community/kernel/src/main/java/org/neo4j/kernel/impl/store/DumpStore.java

```diff
@@ -95,34 +106,37 @@ else if ( !file.isDirectory() && file.getName().indexOf( ':' ) != -1 )
                 throw new IllegalArgumentException( "No such file: " + arg );
             }
         }
-        switch ( file.getName() )
+        try ( NeoStores neoStores = createStoreFactory.apply( file ).openNeoStores( false ) )
```

Comment:
This looks like this now requires all store files. Previously we were able to dump the content of a single store file without needing to have the other store files present.